src/platform/plugins/shared/unified_search/public/search_bar/create_search_bar.tsx

```diff
@@ -282,6 +282,7 @@ export function createSearchBar({
             onESQLDocsFlyoutVisibilityChanged={props.onESQLDocsFlyoutVisibilityChanged}
             esqlEditorInitialState={props.esqlEditorInitialState}
             onEsqlEditorInitialStateChange={props.onEsqlEditorInitialStateChange}
+            hasDirtyState={props.hasDirtyState}
```

Comment:
This is used so that the controls can be hooked up to the unified search refresh button when the new Dashboard setting `autoApplyFilters` is `false`